CheckPkgConfig.py

```diff
@@ -37,7 +37,7 @@ def check_file(self, pkg, filename):
         if pkg.grep(self.suspicious_dir, filename):
             Filter.printError(pkg, "invalid-pkgconfig-file", filename)
 
-        pc_file = open(pkg.dirName() + "/" + filename, "r")
+        pc_file = open(pkg.dirName() + "/" + filename, "r", encoding='utf-8')
```

Comment:
wouldn't we want to have ''', errors='ignore'''' here as well?

Reply:
Yes, as we can't assure that all .pc files are in utf-8, it's our only reasonable option to ignore errors, I guess.